app/ui/preload/index.js

```diff
@@ -15,6 +15,10 @@ import './active-element';
 import Readability from './readability';
 import { readerify } from './reader';
 import { parseMetadata } from './metadata-parsing';
+import { reload } from './reload';
 
 window._readerify = readerify.bind(null, Readability);
 window._parseMetadata = parseMetadata;
+// Expose _TOFINO_RELOAD for the tofino://sessionrestore URLs that need
+// the webview to reload itself
+window._TOFINO_RELOAD = reload;
```

Comment:
Not sure why you'd want to save this as a global on the window, as opposed to just importing the module where it's needed.

For _readerify and _parseMetadata, those functions are exported on the window global since they're accessed in sagas.